docs/en/02_advanced_setup.md

```diff
@@ -50,6 +50,12 @@ class BlockPage extends Page
 }
 ```
 
+> [!IMPORTANT]
```

Comment:
I haven't changed any logic in this PR that would break any sites that programatically call publishSingle() somewhere for something - however it's worth calling out that the intended way for elemental to be run with it's nested data hierarchy is publishRecursive()